ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -20,7 +20,16 @@ import com.daml.platform.common
 import com.daml.platform.common.MismatchException
 import com.daml.platform.configuration.ServerRole
 import com.daml.platform.indexer.parallel.ParallelIndexerFactory
+import com.daml.platform.store.DbType.{
+  AsynchronousCommit,
+  LocalSynchronousCommit,
+  SynchronousCommit,
+}
 import com.daml.platform.store.appendonlydao.events.{CompressionStrategy, LfValueTranslation}
+import com.daml.platform.store.backend.DataSourceStorageBackend.{
+  DataSourceConfig,
+  PgSynchronousCommitValue,
```

Comment:
Yeah this is a bit weird but intentional: I'd like to leave StorageBackend agnostic of the logic what is the desirable configuration for which StorageBackend, and how we make that configurable otherwise.
So that config is part of the interface, it is deliberatly exposed. For example if we want to configure here oracle as well we will have to do that for both.

Reply:
Ack. I thought so once I've read the further code.